controllers/storagecluster/storageconsumer.go

```diff
@@ -24,6 +26,20 @@ type storageConsumer struct{}
 var _ resourceManager = &storageConsumer{}
 
 func (s *storageConsumer) ensureCreated(r *StorageClusterReconciler, storageCluster *ocsv1.StorageCluster) (ctrl.Result, error) {
+	consumerConfigMap := &corev1.ConfigMap{}
+	consumerConfigMap.Name = localStorageConsumerConfigMapName
+	consumerConfigMap.Namespace = storageCluster.Namespace
+	if _, err := controllerutil.CreateOrUpdate(r.ctx, r.Client, consumerConfigMap, func() error {
+		if consumerConfigMap.Data == nil {
+			consumerConfigMap.Data = map[string]string{}
+		}
+		resourceMap := util.WrapStorageConsumerResourceMap(consumerConfigMap.Data)
+		resourceMap.MarkNoobaaAccountNotApplicable()
+		return nil
+	}); err != nil {
+		return ctrl.Result{}, fmt.Errorf("failed to create/update storageconsumer configmap %s: %v", localStorageConsumerConfigMapName, err)
+	}
```

Comment:
This code need to move to the same place where we are already creating the CM for the internal client